src/cli/utils.ts

```diff
@@ -6,23 +6,36 @@ import path from 'path'
 import { DeploymentResult } from './CommandsLogic'
 import { RelayHubConfiguration } from '../relayclient/types/RelayHubConfiguration'
 
-const cliInfuraId = '41a7aa85a67641f3bd6e31cb60753698'
+const cliInfuraId = '$INFURA_ID'
 export const networks = new Map<string, string>([
   ['localhost', 'http://127.0.0.1:8545'],
   ['xdai', 'https://dai.poa.network'],
   ['ropsten', 'https://ropsten.infura.io/v3/' + cliInfuraId],
   ['rinkeby', 'https://rinkeby.infura.io/v3/' + cliInfuraId],
   ['kovan', 'https://kovan.infura.io/v3/' + cliInfuraId],
+  ['goerli', 'https://goerli.infura.io/v3/' + cliInfuraId],
   ['mainnet', 'https://mainnet.infura.io/v3/' + cliInfuraId]
 ])
 
 export function supportedNetworks (): string[] {
   return Array.from(networks.keys())
 }
 
-export function getNetworkUrl (network = ''): string {
-  const match = network.match(/^(https?:\/\/.*)/) ?? []
-  return networks.get(network) ?? match[0]
+export function getNetworkUrl (network: string, env: {[key: string]: string|undefined} = process.env): string {
```

Comment:
Why do you need to get env as parameter here?

Reply:
so I can test it.
we used to have the infuraid burried into the code. which means all requests could use the same quotta.
instead, now it uses env.var INFURA_ID, and using a command-line parameter like `--network kovan` is simple.